dbms/src/Storages/FormatVersion.h

```diff
@@ -130,6 +131,15 @@ inline static const StorageFormatVersion STORAGE_FORMAT_V5 = StorageFormatVersio
     .identifier = 5,
 };
 
+inline static const StorageFormatVersion STORAGE_FORMAT_V6 = StorageFormatVersion{
```

Comment:
We should change `STORAGE_FORMAT_CURRENT` as well.

Reply:
Now we can manually set it by config file and test it with the lightweight ingest index feature. After the feature is  available, then we advance the `STORAGE_FORMAT_CURRENT`.